core/meterer/paymentlogic/paymentlogic.go

```diff
@@ -0,0 +1,160 @@
+package paymentlogic
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"math/big"
+	"slices"
+	"time"
+
+	"github.com/Layr-Labs/eigenda/core"
+)
+
+// GetBinLimit returns the bin limit given the bin interval and the symbols per second
+func GetBinLimit(symbolsPerSecond uint64, binInterval uint64) uint64 {
+	if symbolsPerSecond == 0 || binInterval == 0 {
```

Comment:
Could you add a doc explaining this case? What does it mean to return a zero for this?

Reply:
yes, added in [c680333](https://github.com/Layr-Labs/eigenda/pull/1693/commits/c680333685914cda89280d7046108e1ec27a5980)